client/src/components/pages/jobs/Jobs.tsx

```diff
@@ -1,5 +1,89 @@
-const Jobs = () => {
-  return <>Jobs</>;
+import React from "react";
+import JobCard from "../../UI/card/JobCard";
+import styles from "./Jobs.module.css";
+
```

Comment:
we used `styling` not `styles` for the classes

Reply:
i replaced styles with styling